nautobot/dcim/forms.py

```diff
@@ -3198,8 +3198,8 @@ class CableBulkEditForm(TagsBulkEditFormMixin, StatusModelBulkEditFormMixin, Nau
         initial="",
         widget=StaticSelect2(),
     )
-    label = forms.CharField(max_length=100, required=False)
-    color = forms.CharField(max_length=6, required=False, widget=ColorSelect())  # RGB color code
+    label = forms.CharField(max_length=255, required=False)
+    color = forms.CharField(max_length=255, required=False, widget=ColorSelect())  # RGB color code
```

Comment:
Let's revert the `color` fields to `max_length=6` since they only ever store a `RRGGBB` hex string.